src/socket.rs

```diff
@@ -285,6 +285,51 @@ impl Socket {
         })
         .await
     }
+    pub async fn send(&self, buffer: &[u8]) -> Result<usize, Error> {
```

Comment:
Is there something special about this function?
It looks exactly the same as the `write` function. Would prefer to not have duplication.

Reply:
Oh, my bad. Didn't see write. I'll change it